README.md

```diff
@@ -183,6 +184,14 @@ jobs:
 
 <!-- CONFIGURATION -->
 
+Remember that you can always create a configuration file in your project that it will be picked up by the tool running on the action. For example, if you create any files with the following names, `codeowners-generator` will pick it up:
+
+- a `codowners-generator` property in package.json
+- a `.codowners-generatorrc` file in JSON or YAML format
+- a `.codowners-generator.json`, `.codowners-generator.yaml`, `.codowners-generator.yml`, `.codowners-generator.js`, or `.codowners-generator.cjs` file
+- a `codowners-generatorrc`, `codowners-generator.json`, `codowners-generatorrc.yaml`, `codowners-generatorrc.yml`, `codowners-generator.js` or `codowners-generator.cjs` file inside a .config subdirectory
+- a `codowners-generator.config.js` or `codowners-generator.config.cjs` CommonJS module exporting an object
+
```

Comment:
I feel like parts of this belong under the configuration heading.

Reply:
Yea, maybe moving this block to config and linking the remembering part. 

I just want ppl to be aware of this, because I don't think it was that clear.

Reply:
I think that sounds good, moving the main portion of this to configuration and adding a paragraph about config being picked up from the same places as the CLI tool and a link to the section about the config files.